client.js

```diff
@@ -45,100 +42,338 @@ bot.on('message', function (message) {
+                    message.author.send(`People change.  No longer blocking ${formatCapitalCase(args[0])} on either server.`)
+                }
+                break;
+
+            //TODO:
+            case 'BLOCKS':
+                db.showBlocks(message.author.id, (res) => {
+                    let blocks = [];
+                    if (res.user_blocks.length === 0 && res.watch_blocks.length === 0) {
+                        message.author.send(`You haven't blocked any sellers.  Use \`!block seller, server\` to block a seller on all watches, or react with the \`🔕\` emoji on a watch notification to block a seller only for a certain item.`)
+                    } else {
+                        if (res.user_blocks.length > 0) {
+                            res.user_blocks.forEach(block => {
+                                blocks.push({
+                                    name: `${formatCapitalCase(block.seller)} (${formatCapitalCase(block.server)})`,
+                                    value: `All Watches`,
+                                    inline: false
+                                })
+                            })
+                            
+                        } if (res.watch_blocks.length > 0) {
+                            res.watch_blocks.forEach(block => {
+                                blocks.push({
+                                    name: `${formatCapitalCase(block.seller)} (${formatCapitalCase(block.server)})`,
+                                    value: `\`${formatCapitalCase(block.name)}\` Watch`,
+                                    inline: false
+                                })
+                            })
+                        }  
+                        message.author.send(
+                            new Discord.MessageEmbed()
+                            .setColor('#EFE357')
+                            .setTitle(`__Blocks__`)
+                            .addFields(blocks)
+                        )
+                        .catch(console.error);
+                    }
+                })
+                break;
+            case 'SNOOZE':
+                //snooze all watches
+                if (args && args[0]) {
+                    db.snooze('USER', message.author.id, args[0]) 
+                    message.author.send(`Sleep is good.  Pausing notifications on all watches for ${args[0]} hours.  Use \`\`!unsnooze\`\` to resume watch notifications.`)
+                } else {
+                    db.snooze('USER', message.author.id) 
+                    message.author.send(`Sleep is good.  Pausing notifications on all watches for 6 hours.  Use \`\`!unsnooze\`\` to resume watch notifications.`)
+                }
+                break;
+            case 'UNSNOOZE':
+                //TODO: unsnooze all watches
+                db.unsnooze('USER', message.author.id)
+                message.author.send(`Rise and grind.  Let's get that Loaf of Bread`)
```

Comment:
lolwat

Anyway, missing punctuation if you happen to make another pass.

Reply:
Good catch.